modules/javafx.controls/src/test/java/test/javafx/scene/control/TreeTableViewTest.java

```diff
@@ -1772,6 +1774,49 @@ private void addChildren(TreeItem parent, String name) {
         VirtualFlowTestUtils.assertCellTextEquals(table, 2, "2", "updated name2");
     }
 
+    @Test public void testSetChildrenShouldUpdateTheCells() {
```

Comment:
we probably should have `@Test` on its own line

Reply:
sure, just copied from above.